src/ntttcp.h

```diff
@@ -91,6 +91,7 @@ struct ntttcp_stream_client{
 
 	uint	  num_conns_created;
 	uint64_t  total_bytes_transferred;
+	int*	  sockfds;
```

Comment:
What's the purpose of those dockets? Do you need to establish the connection to server from those sockets?
Note, the real test connections are made from the sockets created inside of the run_ntttcp_sender_tcp_stream().

Reply:
I didn't create additional sockets. These test sockets are just stored in ntttcp_stream_client struct so I can call getsockopt to retrieve RTT after the transfer is done. I didn't change any existing logic. Sockets are now closed after RTT are retrieved.

Reply:
Test sockets were stored in an automatic array in the original code and now they are stored here.